sdk/eventhub/event-processor-host/test/iothub.spec.ts

```diff
@@ -9,22 +9,29 @@ chai.use(chaiAsPromised);
 import debugModule from "debug";
 const debug = debugModule("azure:eph:iothub-spec");
 import {
-  EventPosition, OnReceivedError, PartitionContext, EventData, OnReceivedMessage, EventProcessorHost
+  EventPosition,
+  OnReceivedError,
+  PartitionContext,
+  EventData,
+  OnReceivedMessage,
+  EventProcessorHost
 } from "../src";
 import { delay } from "@azure/event-hubs";
 dotenv.config();
 
-describe("EPH with iothub connection string", function (): void {
+describe("EPH with iothub connection string", function(): void {
```

Comment:
Another option that prettier doesn't support is the space after function keyword in anonymous function.  I could live with it.

There's an open issue tracking support in prettier but it doesn't get much traction. https://github.com/prettier/prettier/issues/3847